### PR TITLE
Update contributors and metadata; regenerate cldf files

### PR DESCRIPTION
**Pull request by tresoldi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# The Internal Structure of Pama-Nyungan
+# CLDF dataset derived from Bowern and Atkinson's "Internal Structure of Pama-Nyungan", from 2012
```
Comment (chrzyki) [inline README.md:1]: No comma after the "TITLE" I think?

Comment (tresoldi) [reply README.md:1]: I though we had decided for a comma after all? I put in in email suggestion, but it was easy to overlook it...

@LinguList , could you confirm?

Comment (SimonGreenhill) [reply README.md:1]: It's not needed.

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # Contributors
 
-Name | GitHub user | Role
---- | --- | ---
-Tiago Tresoldi | @tresoldi | maintainer
-Johann-Mattis List | @lingulist | maintainer
+Name | GitHub user | Description | Role
+--- | --- | --- | --- |
+Tiago Tresoldi | @tresoldi | patron, orthography, concepts | Autor
```
Comment (xrotwang) [inline CONTRIBUTORS.md:5]: "Aut**h**or"

**Pull request merged**